npm ERR! ERESOLVE unable to resolve dependency tree

Is this package abandoned? Would you recommend something else? 



This package looked interesting to me, but I can't even install it because I'm using `Jest 27` and you have this in your `package.json`:
```json
  "peerDependencies": {
    "jest": "25.x || 26.x"
  },
```

### Probable fix:
```json
  "peerDependencies": {
    "jest": ">=25"
  },
```

## Workaround
`npm i --force jest-to-match-shape-of`

### Error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: sol-server@1.0.0
npm ERR! Found: jest@27.3.1
npm ERR! node_modules/jest
npm ERR!   dev jest@"^27.3.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer jest@"25.x || 26.x" from jest-to-match-shape-of@1.3.1
npm ERR! node_modules/jest-to-match-shape-of
npm ERR!   dev jest-to-match-shape-of@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/phil/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/phil/.npm/_logs/2021-10-29T21_37_03_830Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -65,7 +65,7 @@
   "types": "dist/index.d.ts",
   "name": "jest-to-match-shape-of",
   "peerDependencies": {
-    "jest": ">=25x"
+    "jest": ">=25"
   },
   "repository": "git@github.com:Dean177/jest-to-match-shape-of.git",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -65,7 +65,7 @@
   "types": "dist/index.d.ts",
   "name": "jest-to-match-shape-of",
   "peerDependencies": {
-    "jest": "25.x || 26.x"
+    "jest": ">=25x"
   },
   "repository": "git@github.com:Dean177/jest-to-match-shape-of.git",
   "scripts": {

